### ISSUE TITLE
Level: Bomb on the hill

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#499]
```
#########
#   y   #
#########
#Q     (#
     p( #
/  /##\O#
#########
:bash=1
:block=1
:dig=1
:name=Bomb on the hill
:num_rabbits=3
:num_to_save=3
```

**Comment by colonelfazackerley** [comment]
v+]UZYeY|ZznzHz!v]IYqX+DLzE.Hdv.z,,|H

Enjoyable. I had the idea on my second go. Took one more attempt to get the timing.

**Comment by tttppp** [comment]
Not quite the same as colonel's solution:
```)g!^O=^)O=dQtUg3gggx[xc=glJ,Zgd Utgd)t```

**Comment by andybalaam** [comment]
Released in 0.11
